src/Table.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react'
+import { DataGrid } from '@mui/x-data-grid';
+import { useState,useCallback } from 'react';
+import SearchBar from "material-ui-search-bar";
+import axios from 'axios';
+import debounce from 'lodash.debounce'
+
+
+export default function Table_Display() {
+  var [rows, setRows] = useState([]);
+  var [searchfilter, setSearchfilter] = useState([]);
+  const [searched, setSearched] = useState("");
+  let timeout;
+  const columns_countries = [
+    { field: 'name', headerName: 'Countrey Name', width: 350 },
+    { field: 'code', headerName: 'Countrey Code', width: 350 },
+  ];
+
+  const handleSearch = debounce(searchVal =>{
+   
+      setRows(searchfilter.filter((row) => {
+        return row.name.toLowerCase().includes(searchVal.toLowerCase());
+      }));
+      },
+       500);
+  
+  const cancelSearch = () => {
+    setSearched("");
+    setRows(searchfilter);
+
+  }
+  const handleTimeoutSearch=(searchVal) =>timeout=setTimeout(() => {
+    clearTimeout(timeout);
+    handleSearch(searchVal);
+  },1000 ) 
+ 
+  
+
+
+  useEffect(() => {
+    axios.get(`https://gist.githubusercontent.com/keeguon/2310008/raw/bdc2ce1c1e3f28f9cab5b4393c7549f38361be4e/countries.json`, {

```

Comment:
1. use a constant variable from the constants file
2. the constant should be CAPITALIZED
3. use a utility function `import { get } from 'utils/axios'`